packages/larva-patterns/objects/o-card/o-card.prototype.js

```diff
@@ -14,6 +16,8 @@ module.exports = {
 	o_card_content_classes: '',
 	c_lazy_image: c_lazy_image,
 	c_span: c_span,
+	c_tagline: c_tagline,
```

Comment:
Won't this and the addition of `o_author` break backwards compatibility by introducing additional prototype data that isn't expected by existing uses of this object?

Reply:
@tinuola the solution for this is to set c_tagline to false by default, then including it in the `module` that uses it. We need to document this practice somewhere.